Snakefile

```diff
@@ -60,6 +60,21 @@ rule prepare_conv_pp:
     shell:
         "python scripts/prepare_conv_pp.py {input.opsd} {input.gpkg} {input.b3_regions} {input.scalar_template} {output}"
 
+
+rule prepare_heat_demand:
+    input:
+        raw="raw/",
+        dishh="raw/distribution_households.csv",
+        hl="raw/holidays.csv",
+        script="scripts/prepare_heat_demand.py",
+        sc="raw/scalars.csv",
+        tstemp="oemof_b3/schema/timeseries.csv"
```

Comment:
I would advise to give more telling names here, even if they are longer. I would put the script in the end of those.

Reply:
Done with commit https://github.com/rl-institut/oemof-B3/pull/76/commits/f2b37f8082de1b48b5c269947c5a8171703cf94b.